templates/cert-manager/cert-manager.yml

```diff
@@ -0,0 +1,6409 @@
+# This file is copied from:
```

Comment:
How are we going to support this file in CRW? Have we checked that the licenses are compatible? https://github.com/jetstack/cert-manager/blob/master/LICENSES

Also, all these definitions (apart from the CRDs) are managed through helm. Would it not be possible to use helm for installing them instead of hardcoding this humongous file into our codebase?

Reply:
> Have we checked that the licenses are compatible? 

Cert Manager [is licensed](https://github.com/jetstack/cert-manager/blob/master/LICENSES#L2) under Apache License Version 2.0. That mean we can use it.

> Would it not be possible to use helm for installing them

Yes, it is possible. But we plan to use the same approach for operator. And there using Helm will be weird.